lib/src/modules/conditional.dart

```diff
@@ -30,9 +30,43 @@ abstract class _Always extends Module with CustomSystemVerilog {
 
   final Uniquifier _portUniquifier = Uniquifier();
 
-  _Always(this.conditionals, {super.name = 'always'}) {
+  /// Executes provided [conditionals] at the appropriate time (specified by
+  /// child class).
+  ///
+  /// If [reset] is provided, then all signals driven by this block will be
+  /// conditionally reset when the signal is high.
+  /// The default reset value is to `0`, but if [resetValues] is provided then
+  /// the corresponding value
+  /// associated with the driven signal will be set to that value instead upon
+  /// reset.
+  _Always(this.conditionals,
+      {Logic? reset, Map<Logic, dynamic>? resetValues, super.name = 'always'}) {
     // create a registration of all inputs and outputs of this module
     var idx = 0;
+
+    // Get all Receivers
+    final allReceivers =
+        conditionals.map((e) => e.getReceivers()).expand((e) => e).toList();
```

Comment:
Please pull from `main`, there's been some changes in `Conditional`, so you can use `receivers` instead of `getReceivers`, which is an API deprecation that also gives you some perf boost here.